cognite/client/_api/data_modeling/instances.py

```diff
@@ -558,19 +591,34 @@ def _load(
                     node_cls._load(data) if data["instanceType"] == "node" else edge_cls._load(data)
                     for data in resource_list
                 ]
-                return cls(resources, None)
+                return cls(resources, None, None)
+
+            @classmethod
+            def _load_raw_api_response(
+                cls, responses: list[dict[str, Any]], cognite_client: CogniteClient
+            ) -> _NodeOrEdgeList:
+                typing: TypeInformation | None = None
+                if len(responses) >= 1:
+                    typing = TypeInformation._load(responses[0]["typing"]) if "typing" in responses[0] else None
+                resources = [
+                    node_cls._load(data) if data["instanceType"] == "node" else edge_cls._load(data)
```

Comment:
Why don't we have an adapter for this? 😜

Reply:
This is an adapter? Not sure if I understand you here